src/gcinfo/gcinfodumper.cpp

```diff
@@ -300,7 +300,7 @@ PORTABILITY_ASSERT("GcInfoDumper::ReportPointerRecord is not implemented on this
                 pReg = *(SIZE_T**)(pContext + rgRegisters[iReg].cbContextOffset);
                 if (iEncodedReg == iSPRegister)
                 {
-                    pReg = (SIZE_T*)((BYTE*)pRD->pCurrentContext + rgRegisters[iEncodedReg].cbContextOffset);
+                    pReg = (SIZE_T*)((BYTE*)pRD->pCurrentContext + rgRegisters[iReg].cbContextOffset);
```

Comment:
In stead of changing it here should we not be changing the value of iSPRegister in https://github.com/dotnet/coreclr/blob/master/src/gcinfo/gcinfodumper.cpp#L207 to be 1 less ?